tasks/gulp/backstop-config.js

```diff
@@ -69,7 +69,10 @@ module.exports = ({ host, port, components }) => ({
   ],
   onBeforeScript: 'puppet/onBefore.js',
   onReadyScript: 'puppet/onReady.js',
-  scenarios: buildScenarioList(host, port, components),
+  scenarios: buildScenarioList(host, port, components).flatMap((scenario) => [
+    // this means we have twice as many VRT which might increase test time a lot...
+    scenario, { ...scenario, label: `${scenario.label} (using rebrand)`, url: `${scenario.url}?rebrand=true` },
```

Comment:
in the future we might want to reduce this, but for this initial intro of the rebrand it's useful to have a snapshot of all the components with rebrand enabled